src/lib/lessons.ts

```diff
@@ -137,31 +114,3 @@ export async function loadLesson(slug: string, token?: string) {
 
   return lesson as LessonResource
 }
-
-export async function loadLessonForUser(slug: string) {
```

Comment:
This function was only used in one place and that was removed back in
February 2021 and replaced with a call to `loadLesson`.
https://github.com/eggheadio/egghead-next/commit/11767d06554c85de203631c30ad0917cd4d11508